pom.xml

```diff
@@ -39,13 +39,6 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>git</artifactId>
             <version>2.3</version>
-            <exclusions>
-                <!-- To avoid ClassNotFoundException during InjectedTest -->
-                <exclusion>
-                    <groupId>org.apache.httpcomponents</groupId>
-                    <artifactId>httpclient</artifactId>
-                </exclusion>
-            </exclusions>
```

Comment:
If you need this artifact, please add the dependency with the version that you need.